impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -467,17 +483,28 @@ private Response execute(Request request) throws ResourceException {
 
         Assert.notNull(response, "response argument cannot be null.");
 
-        Map<String, Object> out = null;
+        if (response.hasBody() && (MediaType.APPLICATION_JSON.equals(response.getHeaders().getContentType())
+            || APPLICATION_ION_JSON.equals(response.getHeaders().getContentType()))) {
```

Comment:
nit: This is probably fine for now, given that Okta only has app/json and ion endpoints, but... `endsWith("+json")` might be more correct: https://tools.ietf.org/html/rfc6839